CHANGELOG.md

```diff
@@ -54,6 +54,18 @@ Other Changes
 - Lepton was upgraded to the latest version (PR #349)
 - Made Object::print a const member function (PR #191)
 - Improved the testOptimization/OptimizationExample to reduce the runtime (PR #416)
+- Added shouldBeParallelIfPossible() override methods to all computationally
+expensive muscle force classes
```

Comment:
> I am also wary about using this method to also indicate the force computation is not threadsafe. 

All the force elements should be assumed *not* to be threadsafe. It takes considerable effort and expertise to make a safe one, as we've seen with muscles here. Safe means two things: (1) multiple copies of this force element can run in parallel, and (2) this force element promises not to break any of the non-threadsafe forces. And, only expensive computations are worth parallelizing. So we need to collect from the force element author enough information to tell the force subsystem whether it is (a) safe, and (b) desirable to parallelize a particular force element. It was hard to come up with a good name that captures the right information. Do you have a better idea for the name or process?